### ISSUE TITLE
xdpyinfo:  unable to open display ":99". and xvfbMaxStartWaitTime

### ISSUE THREAD

**Issue by Lewik** [maptiler/tileserver-gl#386]
I am using tileglserver on docker (klokantech/tileserver-gl)

Trouble:
multiple `xdpyinfo:  unable to open display ":99".` and then `Could not connect to display 99 in 5 seconds time.`

Solution:
copy run.sh from this repo and change xvfbMaxStartWaitTime to 60 seconds (seems my server needs 20 seconds to start xvfb). After that use docker volume to replace run.sh. 
Example for docker-compose
```
    volumes:
      - "./run.sh:/usr/src/app/run.sh"
```

https://github.com/maptiler/tileserver-gl/commit/8e1071aad038b960767f1910d6ea86974496b2e4
@guenhter 

5 seconds is hot enough for some machines.
There is interesting comment in run.sh:
`# Wait to be able to connect to the port. This will exit if it cannot in 15 minutes.` (but there are 5 seconds only)

There is two ways: increase hardcoded timeout or provide the way to set timeout from outside

**Comment by qubiack** [comment]
After I change a code as you were sad, it doesn't change anything for me. I all time get info:

```
maps_1  | Starting Xvfb on display 99
maps_1  | xdpyinfo:  unable to open display ":99".
maps_1  | 
maps_1  | Starting tileserver-gl v3.0.0
maps_1  | Using specified config file from config.json
maps_1  | Starting server
maps_1  | Listening at http://[::]:80/
maps_1  | Startup complete
```

But when I add a `sleep 1` just after:
```
start-stop-daemon --start --pidfile ~/xvfb.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :${displayNumber} -screen ${screenNumber} 1024x768x24  -ac +extension GLX +render -noreset
```

I get different response:
```
maps_1  | Starting Xvfb on display 99
maps_1  | 
maps_1  | Starting tileserver-gl v3.0.0
maps_1  | Using specified config file from config.json
maps_1  | Starting server
maps_1  | Listening at http://[::]:80/
maps_1  | Startup complete
```

But my maps raster maps doesn't work at all.

**Comment by lhsm** [comment]
Could you update the docker image with this fix, please?

**Comment by patrickdickey52761** [comment]
I installed the tileserver-gl and its associated docker components today and still received this error. What needs to happen to get this updated in the repository, so we're able to get by this? Or, do I need to clone a development branch in order to see the changes? The command that I used was sudo docker run --rm -it -v $(pwd):/data -p 8080:80 klokantech/tileserver-gl

Thank you and have a great day. :)
Patrick.
